set.mm

```diff
@@ -28607,6 +28610,7 @@ generally appear in a single form (either definitional, but more often
       ( cv wcel wi ex a2d ralimdv2 ) ABCDEEADGEHZBCAMBCIFJKL $.
   $}
 
+
```

Comment:
extra line

Now that I think about it, ~ conventions doesn't say anything about whitespace, the only thing is [`CONTRIBUTING.md`: following metamath.exe rewrap](https://github.com/metamath/set.mm/blob/develop/CONTRIBUTING.md#formatting-recommendation-prior-to-submitting-a-pull-request), which doesn't detect this. The usual spacing seems to be one line between items like theorems or blocks, and two lines at the end of each section, but there are plenty of counterexamples...

Reply:
Fair enough. I'll touch this up for now and leave the conversation for another day...

Reply:
At some point, we agreed on the rule "double blank lines if and only if they separate sections". This rule is globally followed in set.mm and iset.mm.  But this is not done by the automatic rewrapping (wouldn't be too hard to do, probably), so we do not enforce it.